riscv/riscv.mk.in

```diff
@@ -1329,7 +1329,6 @@ riscv_insn_list = \
 	$(if $(HAVE_INT128),$(riscv_insn_ext_v),) \
 	$(riscv_insn_ext_zce) \
 	$(riscv_insn_ext_h) \
-	$(riscv_insn_ext_p) \
```

Comment:
This commit also needs to disallow P in `isa_parser.cc`, so that nobody is confused about whether P is supported or not.